### DISCUSSION TITLE
Why highlight search closed after move cursor or page up/down

### DISCUSSION THREAD

**Discussion by rickywu** [ayamir/nvimdots#821] [General]
I use search highlight, it dispear after move cursor or page up/down
sometimes can keep if use hjkl to move and keep for few pages when use page up/down

**Comment by ayamir** [comment]
Disable this plugin:
https://github.com/ayamir/nvimdots/blob/f1125a96acc769a21a7443ae1812dfdfe81812de/lua/modules/plugins/editor.lua#L70-L73
